### ISSUE TITLE
Feature Request: Show channel info in Thumbs

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#528]
When I have the same image open in different channels, it is quite impossible to tell which thumbnail is in which channel until I click on one and see which channel gets focused. I know it is possible to find out by just switching to `Contents` but I wonder if there is a way to have `Thumbs` tooltip also display channel the channel a thumbnail belongs to.

If this is *already* possible, e.g., via `plugin_Thumbs.cfg`, please tell me how. Thanks!

**Comment by ejeschke** [comment]
I've been thinking the same thing since I started working on #499.  I think we could pretty easily show the channel name in the popup tool tip.  What do you think, @pllim?

**Comment by pllim** [comment]
Yes, I agree. But maybe when an option to turn it off because there's always someone who somehow does not want it.

**Comment by ejeschke** [comment]
Do you use the `tt_keywords` setting of `Thumbs` to customize `stginga`'s pop-ups?

**Comment by pllim** [comment]
Yes, I use them.
